bson/bsoncodec/default_value_decoders_test.go

```diff
@@ -1322,7 +1322,11 @@ func TestDefaultValueDecoders(t *testing.T) {
 					nil,
 					&bsonrwtest.ValueReaderWriter{BSONType: bsontype.String, Return: string("not-valid-%%%%://")},
 					bsonrwtest.ReadString,
-					errors.New("parse not-valid-%%%%://: first path segment in URL cannot contain colon"),
```

Comment:
Seems like something change about how `url.Error` stringifies itself in Go 1.14, so this test was failing locally for me after upgrading. Creating a `url.Error` here rather than using `errors.New` fixes the issue and the test passes on both Go 1.13 and Go 1.14.